pages/linux/pdftoppm.md

```diff
@@ -0,0 +1,24 @@
+# pdftoppm
+
+> Convert PDF document pages to Portable Pixmap (image formats).
+> More information: <https://linux.die.net/man/1/pdftoppm>.
+
+- Specify the first page to convert:
+
+`pdftoppm -f`
```

Comment:
Should there be a pdf document which we supply ?
eg like `pdftopm -f {{file.pdf}}`